documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -2617,13 +2617,12 @@ The property is applied to all new caches, it means for all Entities. So propert
 The annotations are applied to the new caches related to the Entities on which they are present.
 So annotations have a local (Entity) scope instead.
 
-Is it possible enabling both the configurations, whenever are both present the templates defined by annotation
-will **override** the template defined by property.
+Is it possible enabling both the configurations, whenever are both present the cache configurations defined by annotation
+will **override** the cache configuration defined by property.
```

Comment:
I'm going to rephrase some of these to make it clearer. Also I wouldn't abuse of syntax highlighting when precise language should be enough.